### PR TITLE
[AR] translated 02.md 

### PR DESCRIPTION
**Pull request by abogaziah**

### CODE REVIEW

File: docs/ar/week02/02.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+
```
Comment (Atcold) [inline docs/ar/week02/02.md:1]: Please, remove this empty line.

File: docs/ar/week02/02.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+title: أسبوع 2
+---
+
+<!--
+## Lecture part A
+
+We start by understanding what parametrised models are and then discuss what a loss function is. We then look at Gradient-based methods and how it's used in the backpropagation algorithm in a traditional neural network. We conclude this section by learning how to implement a neural network in PyTorch followed by a discussion on a more generalized form of backpropagation.
+-->
+
+## الجزء أ من المحاضرة
```
Comment (Atcold) [inline docs/ar/week02/02.md:14]: Is it possible to use "A"?

File: docs/ar/week02/02.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+ سنبدأ بفهم ما هي `النماذج ذات العوامل` ثم سنناقش ما هي `دالة الفرق`. بعد ذلك سننظر في الخوارزميات القائمة علي `الإنحدار` وطريقة إستخدامها في خوارزمية `الانتشار الخلفي` علي `شبكة عصبية` تقليدية.
+ ثم سننهي هذا الجزئ بتعلم طريقة تنفيذ شبكة عصبية من خلال __ ثم مناقشة .ضيغة اخري من `الانتشار الخلفي` اكثر شمولا من السابقة
+
+<!--
+## Lecture part B
+
+We begin with a concrete example of backpropagation and discuss the dimensions of Jacobian matrices. We then look at various basic neural net modules and compute their gradients, followed by a brief discussion on softmax and logsoftmax. The other topic of discussion in this part is Practical Tricks for backpropagation.
+-->
+
+## الجزء ب من المحاضرة
```
Comment (Atcold) [inline docs/ar/week02/02.md:25]: Is it possible to use "B"?

File: docs/ar/week02/02.md (hunk 2 above)
Comment (abogaziah) [reply docs/ar/week02/02.md:13]: it will revert the Arabic string around the English one, it's going to be like:  of the lecture A part

Comment (abogaziah) [reply docs/ar/week02/02.md:13]: it will revert the Arabic string around the English one, it's going to be like:  of the lecture A part

**Pull request merged**